http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -701,8 +779,12 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
           )
       }
 
+      createSecondDisk = if (msg.appType == Galaxy) {
+        createDiskOnlyInGoogle(msg.project, ZoneName("us-central1-a"), msg.appName)
```

Comment:
Maybe we want `adaptError` similar to above, to ensure this falls through to `handleKubernetesError`